ga4gh/datamodel/variants.py

```diff
@@ -415,6 +436,27 @@ def convertVariant(self, record, callSetIds):
             sampleIterator += 1  # REMOVAL
         return variant
 
+    def getVariant(self, compoundId):
+        if compoundId.chrom in self._chromFileMap:
+            varFileName = self._chromFileMap[compoundId.chrom]
+        else:
+            raise exceptions.ObjectNotFoundException(compoundId)
+        pos = int(compoundId.pos)
+        # fetch pos is exclusive, not inclusive
```

Comment:
We should need to do this --- we use 0-based ``start`` elsewhere and it works. I think we just need to change the ``variant.id`` definition. (Which we must do anyway, to add in the hash, right?)